cinema/management/commands/wait_for_db.py

```
@@ -0,0 +1,21 @@
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+import time
+
+
+class Command(BaseCommand):
+    help = "Waits for database to be available"
+
+    def handle(self, *args, **options):

```

Robustness improvement: the loop is infinite with no retry limit or timeout. Add a max attempts counter and, if exceeded, raise `CommandError` so the process fails fast and is easier to diagnose in CI. This also aligns with checklist best-practices for `wait_for_db`. Consider importing `from django.core.management import CommandError` and raising `CommandError("...")` after max retries. See handle() start at line 11.